### DISCUSSION TITLE
tutorials/api/overview/

### DISCUSSION THREAD

**Discussion by giscus** [go-gost/gost#410] [General]
# tutorials/api/overview/

https://gost.run/tutorials/api/overview/

**Comment by bili-jing** [comment]
设置了认证账户之后，api调用的时候，认证账户在哪里填写呢。

**Comment by xx19870829** [comment]
gost -api :18080
flag provided but not defined: -api
报错没有api参数？什么版本有api参数

**Reply by Galiathuss** [reply]
冒号前有个空格

**Reply by ginuerzh** [reply]
v3版本中才支持API。
